tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -127,8 +130,11 @@ def test_bad_file_names(self):
                                      self.folder_prefix)
         self.assertCountEqual(expected_warnings, r['warnings'])
 
+    @mock.patch('validation.main.all_required_files_loaded')
```

Comment:
I noticed that `all_required_files_loaded` was not called during `validate_submission`, I think we can remove this mock

Reply:
I agree, looks like it only gets called in `generate metrics`